src/index.html

```diff
@@ -9,81 +9,48 @@
   <body>
     <h1>List of employees</h1>
     <ul>
-      <li
-        data-position="Accountant"
-        data-salary="$162,700"
-        data-age="33"
-      >
+      <li data-position="Accountant" data-salary="$162,700"
+       data-age="33">
```

Comment:
![image](https://user-images.githubusercontent.com/70009205/210413304-5ca954d7-fb46-4b90-9a2f-d62d15b08211.png)
Revert the previous solution. It was true. Fix it everywhere

![image](https://user-images.githubusercontent.com/70009205/210413491-f0433d61-253f-4dc4-b50a-ac8111a2edb7.png)